.coderabbit.yaml

```
@@ -0,0 +1,18 @@
+  poem: false
+  review_status: true
+  collapse_walkthrough: true
+  auto_review:
+    enabled: true
+    drafts: false
+  tools:
+    languagetool:
+      enabled: true
+      level: default
+chat:
+  auto_reply: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)